provisioner/group_vars/all/vpc_rules.yml

```diff
@@ -60,6 +60,26 @@ workshops:
         from_port: 514
         cidr_ip: 0.0.0.0/0
         rule_desc: WinRM
+      - proto: tcp
+        to_port: 5432
+        from_port: 5432
+        cidr_ip: 0.0.0.0/0
+        rule_desc: Cluster option DB port
+      - proto: tcp
+        to_port: 4369
+        from_port: 4369
+        cidr_ip: 0.0.0.0/0
+        rule_desc: Cluster option RabbitMQ
```

Comment:
3.7 no longer uses RabbitMQ, but even so I wouldn't open it to the world.  If I recall for early versions of the workshop that used completely wide open firewall rules, RabbitMQ was causing the boxes to get exploited.  Better safe than sorry.

Instead I would modify the rule to allow only internal IPs, and modify the tower setup inventory to reference the internal IPs also.

Reply:
Updated to use internal IPs

Reply:
Do we still need the rules if we only use internal IPs?

Reply:
@termlen0 See above, do we really need these rules publicly accessible to everyone?

Reply:
Agree. Removed these rules.